flow/layers/container_layer.cc

```diff
@@ -194,6 +194,9 @@ void ContainerLayer::TryToPrepareRasterCache(PrerollContext* context,
       context->raster_cache &&
       SkRect::Intersects(context->cull_rect, layer->paint_bounds())) {
     context->raster_cache->Prepare(context, layer, matrix);
+  } else {
+    // Don't evict raster cache entry during partial repaint
```

Comment:
I don't think there is a way to distinguish list item getting scrolled offscreen from other partial updates. All we have is a clip rect. The question is - should blinking cursor cause cache eviction at all? If so, it can be implemented. i.e. evict items not painted past X frames.